### PR TITLE
Update french translation for account console

### PR DESCRIPTION
**Pull request by vanrar68**
Add missing translations

### CODE REVIEW

**Comment by vanrar68** [comment]
@spechey can you have a look ?

**Comment by Spechey** [comment]
Sounds good

File: themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_fr.properties [hunk 1]
```diff
@@ -12,9 +12,19 @@ password-display-name=Mot de passe
...
+passwordless=Authentification Sans Mot de Passe
 otp-display-name=Application d''authentification
 otp-help-text=Entrez un code de v\u00e9rification \u00e0 usage unique fourni par l''application d''authentification.
+webauthn-display-name=Cl\u00e9 de S\u00e9curit\u00e9
+webauthn-help-text=Utilisez votre cl\u00e9 de s\u00e9curit\u00e9 pour vous authentifier.
+webauthn-passwordless-display-name=Cl\u00e9 de S\u00e9curit\u00e9
+webauthn-passwordless-help-text=Utilisez votre cl\u00e9 de s\u00e9curit\u00e9 pour une authentification sans mot de passe.
 notSetUp={0} non configur\u00e9(e).
+remove=Supprimer
+refreshPage=Rafraichir la page
```
Comment (ulrich) [inline themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_fr.properties:24]: **Rafraichir** should be **Rafra\u00eechir** (Rafraîchir)

**Review by ulrich — CHANGES_REQUESTED** [review]

File: themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_fr.properties (hunk 1 above)
Comment (vanrar68) [reply themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_fr.properties]: Done. Nice catch @ulrich !

**Review by stianst — APPROVED** [review]

**Pull request merged**